### PR TITLE
Fix CI

### PR DESCRIPTION
**Pull request by arthurp**

### CODE REVIEW

**Comment by arthurp** [comment]
@ddn0 Can you set all the CI jobs, except the MacOS ones, as being required to merge? Just like enterprise. I'd do it, but I don't think I can because I'm not an admin.

**Comment by arthurp** [comment]
Oh and set review required too. It would let me merge without review right now.

**Comment by tylershunt** [comment]
I did it!

**Review by tylershunt — APPROVED** [review]
LGTM - workflow cancelation is only important when we're trying to prevent long queues on our private runners anyway.

**Comment by arthurp** [comment]
> LGTM - workflow cancelation is only important when we're trying to prevent long queues on our private runners anyway.

I would like workflow cancellation. There are queues for the Github runners too. But there isn't a secure way to do it that I can see.

File: .github/workflows/python.yaml [hunk 1]
```diff
@@ -68,7 +66,11 @@ jobs:
       run: |
         brew install ccache curl
       # TODO: Does brew install need bzip2, libz-dev, and make?
-    
+
```
Comment (ddn0) [inline .github/workflows/python.yaml:69]: The TODO above seems orphaned.

**Review by ddn0 — APPROVED** [review]

File: .github/workflows/python.yaml (hunk 1 above)
Comment (arthurp) [reply .github/workflows/python.yaml:69]: I'm gonna do some CI clean up once I have it working. I'll get this then.

**Pull request merged**